tools/libraries/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@airswap/libraries",
-  "version": "4.0.5-beta.6",
+  "version": "4.0.5-beta.7",
```

Comment:
Also need to update the `@airswap/types` dep version below.

Reply:
done